Assignment2/mdelucasg.ttl

```diff
@@ -0,0 +1,13 @@
+@base <https://www.oeg-upm.net/Organization#>
+
+:Class01 			rdfs:includes :Sensor029;
```

Comment:
Properties cannot be prefixed as rdfs: They should use a different prefix, including the base if you want